CHANGELOG.md

```diff
@@ -10,6 +10,24 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ### Added
 
+### Changed
+
+### Deprecated
+
+### Removed
+
+### Fixed
+
+### Security
+
+## [1.0.0-RC1] / 0.21.0 - 2021-06-18
+
+With this release we are introducing a split in module versions.  The tracing API and SDK are entering the `v1.0.0` Release Candidate phase with `v1.0.0-RC1`
```

Comment:
How about changing the tag name to `v1.0.0-rc.1`
This is the format that SemVer uses itself. 
Example: https://semver.org/spec/v2.0.0-rc.1.html

Or at least `v1.0.0-rc1` - I have never seen the capitals `RC` in tags before 😉

Reply:
This was discussed in the SIG meeting and it is valid semver accepted by the Go module tooling.  There's not a strong reason to change it.  Plus, RC is an initialism and Go conventions dictate it be in all caps :)

Reply:
OK😉 

I also saw such patterns under: https://blog.golang.org/publishing-go-modules
And https://golang.org/ref/mod also has lowercase.
But it is not anything critical. Just a matter of taste 😄